pkg/framework/gatherer/oc-cli.go

```diff
@@ -0,0 +1,240 @@
+package gatherer
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// CLI provides function to call the OpenShift CLI, which is using to simplify state gathering during tests
+type CLI struct {
```

Comment:
Is this copied from origin? Wonder if we can just import it? Or maybe move it to a library that's easier to import?

Reply:
I tried to import this thing from origin, but this brings A LOT of new dependencies and plenty of complications with wrangling this.

>Or maybe move it to a library that's easier to import?

For now, this thing doesn't tie much to other code around and is more-less independent. We could move this file out to a separate module if will we need to, but I don't think that it's necessary right now.

Reply:
A comment mentioning where this code came from would be useful I think